services/gitlab.service

```diff
@@ -9,6 +9,7 @@ TimeoutStartSec=0
 TimeoutStopSec=15
```

Comment:
Same thing, gitlab http is contacted via Docker DNS on the protonet bridge network, this does not really care about the gitlab network